### ISSUE TITLE
Error: Prompt [undefined] not found in history

### ISSUE THREAD

**Issue by jelling** [StableCanvas/comfyui-client#3]
**Describe the bug**

Using the example `main.ts` script with the `min.workflow.json` does not return a prompt id, but instead:

```
RunPrompt does not return a prompt id but also does not error
```

And when I log the response to console it shows:

```
{ exec_info: { queue_remaining: 0 } }
```

But sending the same prompt json via a rest client works, returning:
```
{
  "prompt_id": "c445e907-9cd9-48f0-8662-e66bd4d63e5b",
  "number": 12,
  "node_errors": {}
}
```

**To Reproduce**

```
  const client = new ComfyUIApiClient({
    api_host: '{removed}',
    api_base: '',
    sessionName: '',
    // user: "comfy-client",
    user: 'undefined',
    WebSocket,
    fetch: fetch as any,
  });
  client.init();

  const resp = await client.runPrompt(mainWorkflow.prompt, {
    //   workflow: mainWorkflow.workflow,
    disable_random_seed: false,
  });
```

**Additional context**

Here is the`workflow.min.json` I used; the only modification is the checkpoint name. But again, that works fine when sent via a rest client.

```
{
  "prompt": {
    "1": {
      "class_type": "CheckpointLoaderSimple",
      "inputs": {
        "ckpt_name": "wildcardxXLTURBO_wildcardxXLTURBOV10.safetensors"
      }
    },
    "2": {
      "class_type": "CLIPTextEncode",
      "inputs": {
        "text": "best quality, 1girl",
        "clip": ["1", 1]
      }
    },
    "3": {
      "class_type": "CLIPTextEncode",
      "inputs": {
        "text": "worst quality, bad anatomy, embedding:NG_DeepNegative_V1_75T",
        "clip": ["1", 1]
      }
    },
    "4": {
      "class_type": "EmptyLatentImage",
      "inputs": {
        "width": 512,
        "height": 512,
        "batch_size": 1
      }
    },
    "5": {
      "class_type": "KSampler",
      "inputs": {
        "seed": 2765233096,
        "steps": 35,
        "cfg": 4,
        "sampler_name": "dpmpp_2m_sde_gpu",
        "scheduler": "karras",
        "denoise": 1,
        "model": ["1", 0],
        "positive": ["2", 0],
        "negative": ["3", 0],
        "latent_image": ["4", 0]
      }
    },
    "6": {
      "class_type": "VAEDecode",
      "inputs": {
        "samples": ["5", 0],
        "vae": ["1", 2]
      }
    },
    "7": {
      "class_type": "SaveImage",
      "inputs": {
        "filename_prefix": "from-sc-comfy-ui-client",
        "images": ["6", 0]
      }
    }
  }
}
```

**Comment by zhzLuke96** [comment]
fixed a4d94017aff9eb92720a8578689feb1d2263e885

**Comment by jelling** [comment]
I appreciate the quick response and the great codebase. That may have been part of my issue but my ultimate problem was that I did not have `ssl: true` in the client config.
